### ISSUE TITLE
🐛 [BUG]: incorrect filenames in package.json prevents @vue-flow/resize-rotate-node from being imported

### ISSUE THREAD

**Issue by TheOneTrueLX** [bcakmakoglu/vue-flow#364]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

@vue-flow/resize-rotate-node fails to load in vite with the following error:

`10:14:30 AM [vite] Internal server error: Failed to resolve entry for package "@vue-flow/resize-rotate-node". The package may have incorrect main/module/exports specified in its package.json.`

### Expected Behavior

Expected behavior would be for module to load without error in vite.

### Steps To Reproduce

1. add the @vue-flow/resize-rotate-node module to a project
2. attempt to use it in any .vue component

I attempted this in both an existing project and a newly created vue3/vite project and observed the same behavior.

### Relevant log output

```shell
10:14:30 AM [vite] Internal server error: Failed to resolve entry for package "@vue-flow/resize-rotate-node". The package may have incorrect main/module/exports specified in its package.json.
  Plugin: vite:import-analysis
  File: D:/devel/scc-ng/frontend/src/views/OverlayEditorView.vue
      at packageEntryFailure (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:34839:11)
      at resolvePackageEntry (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:34836:5)
      at tryNodeResolve (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:34584:20)
      at Context.resolveId (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:34357:28)
      at async Object.resolveId (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:41031:32)
      at async TransformContext.resolve (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:40800:23)
      at async normalizeUrl (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:37581:34)
      at async TransformContext.transform (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:37720:47)
      at async Object.transform (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:41103:30)
      at async loadAndTransform (file:///D:/devel/scc-ng/frontend/node_modules/vite/dist/node/chunks/dep-6b3a5aff.js:37365:29) (x2)
```

### Anything else?

The issue appears to be due to incorrect filenames listed in package.json:

  "main": "./dist/vue-flow-resize-rotate-node.cjs.js",
  "module": "./dist/vue-flow-resize-rotate-node.es.js",
  "unpkg": "./dist/vue-flow-resize-rotate-node.iife.js",
  "jsdelivr": "./dist/vue-flow-resize-rotate-node.iife.js",

The actual filenames in ./dist are named resize-rotate-node.*.js.  Manually changing the "main", "module", "unpkg", and "jsdelivr" entries to reference the correct filenames in package.json fixes the issue and allows the module to be properly imported.

**Comment by bcakmakoglu** [comment]
Thanks for pointing that out, I'll fix that asap :)

**Comment by bcakmakoglu** [comment]
Should be fixed with `v1.0.1` of resize-rotate-node pkg. If not let me know :)
